frontend/test/metabase/scenarios/question/notebook.cy.spec.js

```diff
@@ -563,7 +563,7 @@ describe("scenarios > question > notebook", () => {
   });
 
   // TODO: add positive assertions to all 4 tests when we figure out implementation details
```

Comment:
Do we want/need to add this?

I.e. Is there any point in actually clicking the "Done" button, and then clicking on "Visualize" to make sure everything works?